pkg/cmd/util/clientcmd/factory.go

```diff
@@ -170,7 +170,7 @@ func NewFactory(clientConfig kclientcmd.ClientConfig) *Factory {
 		clients:               clients,
 	}
 
-	w.Object = func() (meta.RESTMapper, runtime.ObjectTyper) {
+	w.Object = func(bool) (meta.RESTMapper, runtime.ObjectTyper) {
```

Comment:
> We're going to have to start enabling 3rd party resources soon.

Not during the rebase and preferably not like they've done.  We should make a complete caching discovery client upstream, the right code to use a cache unless it fails on an unknown type.  A discovery `RESTMapper` ought not be hard to build either.